ui/pages/swaps/import-token/import-token.stories.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import ImportToken from './import-token';
+
+export default {
+  title: 'Import Token',
+};
+
+export const ImportTokenComponent = () => {
+  const data = {
+    iconUrl: './BAT_icon.svg',
```

Comment:
This 404's -- is it needed?  Not a blocker for this.

Reply:
Its probably due to the static dir for storybook assets, which is addressed in another one of these PRs